psl/builtin-connectors/src/lib.rs

```diff
@@ -32,14 +31,15 @@ pub const MSSQL: &'static dyn Connector = &mssql_datamodel_connector::MsSqlDatam
 pub const MONGODB: &'static dyn Connector = &mongodb::MongoDbDatamodelConnector;
 pub static PLANETSCALE_SERVERLESS: &'static dyn Connector = &planetscale::PLANETSCALE_SERVERLESS;
 pub static NEON_SERVERLESS: &'static dyn Connector = &neon::NEON_SERVERLESS;
+pub static PG_JS: &'static dyn Connector = &pg_js::PG_JS;
```

Comment:
Why add back the boilerplate here?

Reply:
So that it's identical to how we imported `PLANETSCALE_SERVERLESS` and `NEON_SERVERLESS`. Doing the same task in a non-unified way could potentially be a source of confusion, wouldn't you agree? I don't mind how we re-export `PLANETSCALE_SERVERLESS`, `NEON_SERVERLESS`, `PG_JS`, as long as we're consistent with how we do it